HelpSource/Classes/Float.schelp

```diff
@@ -65,3 +90,67 @@ pi.asStringPrec(6)
 7.4.asStringPrec(5)
 7.4.asStringPrec(50)
 ::
+
+subsection::Using Floats as replacement for Integers
+
+In SuperCollider, Floats are 64-bit wide. Because  an link::Classes/Integer:: is only 32-bit, it can capture only whole numbers up to code::-2147483648 .. +2147483647::, which is code::2^31:: (one bit is for the sign), or about code::2 x 10^9::.
+
+Therefore, in some situations it can be useful to calculate with floats also when only whole numbers are needed. You can use 64-bit floats for integer calculations up to code::± 9007199254740992:: (code::2^53::, or about code::9 x 10^15::). Note that you still may get floating point errors when you apply certain operations.
```

Comment:
I think you should remove the last sentence unless you can provide a more concrete explanation.

Reply:
yes.